### PR TITLE
feat: [MH-000] inner layout + some layout changes

### PR DESCRIPTION
**Pull request by EvgeniyKvasyuk**
## Details

InnerBaseLayout (3 columns was added) + some fixes in layout

| Details                     | Description                                      |
| --------------------------- | ------------------------------------------------ |
| JIRA ticket                 | https://marathon-health.atlassian.net/browse/PT- |
| Design source               | N/A                                              |
| Related PRs                 | N/A                                              |
| Documentation update needed | No                                               |
| QA sign-off by              | N/A                                              |

## Changelog

- 

## Checklist

- [x] [Commit message convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [x] [Branch naming convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [ ] Performed self review & removed the debugging code (debugger, console log etc).
- [ ] Updated/Added tests.
- [ ] Added explanatory comment for tricky code explaining "why".
- [x] Tested on major browsers (Chrome, Firefox, Safari, Edge)
- [x] Manual testing or integration testing done.
- [x] Added strings via locale

## Screenshots / Animated GIFs

| Before | After |
| ------ | ----- |

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -116,6 +116,7 @@ module.exports = {
     'no-restricted-globals': 'warn',
     'react/no-array-index-key': 'warn',
     'react/jsx-curly-brace-presence': 'warn',
+    'react/function-component-definition': 'off',
```
Comment (EvgeniyKvasyuk) [inline .eslintrc.js:119]: I think it is ok to create a Functional Component with an arrow function. What do you think, guys?

File: src/components/InnerLayout/InnerLayout.module.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  @media (max-width: $screenMobileMax) {
+    flex-direction: column;
+  }
+
+  &__left-sidebar {
+    flex: 1 1 auto;
+    box-sizing: border-box;
+    max-width: 25.625rem;
+    margin-right: 1rem;
```
Comment (VladSolodjankin) [inline src/components/InnerLayout/InnerLayout.module.css:24]: why right sidebar and left sidebar have different margins?

File: src/components/InnerLayout/InnerLayout.module.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  @media (max-width: $screenMobileMax) {
+    flex-direction: column;
+  }
+
+  &__left-sidebar {
+    flex: 1 1 auto;
+    box-sizing: border-box;
+    max-width: 25.625rem;
```
Comment (VladSolodjankin) [inline src/components/InnerLayout/InnerLayout.module.css:23]: I think we can create constant for width

File: src/components/InnerLayout/InnerLayout.module.css (hunk 2 above)
Comment (EvgeniyKvasyuk) [reply src/components/InnerLayout/InnerLayout.module.css:24]: It was 0:30 after an assessment day) what did you look for from me?)

File: src/components/InnerLayout/InnerLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  leftSidebar?: ReactElement;
+  rightSidebar?: ReactElement;
+  className?: string;
+};
+
+export function InnerLayout({ content, leftSidebar, rightSidebar, children, className }: InnerLayoutProps) {
+  return (
+    <div className={cn('inner-layout', className)}>
+      {leftSidebar && <div className={cn('inner-layout__left-sidebar')}>{leftSidebar}</div>}
+      <div className={cn('inner-layout__content')}>{content || children}</div>
```
[the remark this answers is not in the document]
Comment (VladSolodjankin) [reply src/components/InnerLayout/InnerLayout.tsx:21]: lets use children

**Pull request merged**